ui/src/alerting/actions/checks.ts

```diff
@@ -168,7 +173,7 @@ export const updateCheck = (check: Partial<Check>) => async (
 }
 
 export const deleteCheck = (checkID: string) => async (
-  dispatch: Dispatch<Action | NotificationAction>
+  dispatch: Dispatch<any>
```

Comment:
Do we get anything by typing `dispatch`? Feels like it's more trouble than it's worth.

Reply:
We ensure that the actions are of the correct types.

Reply:
It's been awesome for me.  It ensures that the actions we pass dispatc conform to the Action generic.